internal/cli/output_opts.go

```diff
@@ -105,8 +113,8 @@ func (opts *OutputOpts) Print(v interface{}) error {
 }
 
 // parseTemplate will try to find if the given format is a user given template, either by string or file and use it.
-// Current available user templates are  "go-template=Template string" and "go-template-file=path/to/template"
-func (opts *OutputOpts) parseTemplate() (string, error) {
+// Current available user templates are  "go-template=Template string", "go-template-file=path/to/template" and "json-path=path"
+func (opts *OutputOpts) parseTemplate() (outputTemplate, v string, e error) {
```

Comment:
[super-nit] this method is called `parseTemplate` but json path is not technically a template, you could rename this to make the intention clearer or a better option in my opinion would be to split this in two methods, one to get the output type and value, another to read from file if file is given, I think we only error here because of a file and that would bring back the returned value per method to two